lib/facility.rb

```diff
@@ -1,14 +1,52 @@
 class Facility
   attr_reader :name, :address, :phone, :services
 
-  def initialize(name, address, phone)
+  def initialize(name, address, phone, services)
```

Comment:
So we don't need to add services here in the argument list since we're making it a default on line 8. Also, line 8 should be what you had originally @services = []